htdocs/install/mysql/tables/llx_stocktransfer_stocktransfer.key.sql

```diff
@@ -0,0 +1,29 @@
+-- Copyright (C) ---Put here your own copyright and developer email---
```

Comment:
With v16, new tablesare now installed only when module is enabled, so you must:
- remove any code to create table into migration (no more need).
- rename your table llx_aaa_bbb.sql  into llx_aaa_bbb-modulename.sql
- you must include the lines at begin of method public function init($options = '') in you rmodule descriptor file :
 		$result = $this->_load_tables('/install/mysql/', 'mymodule');
		if ($result < 0) {
			return -1; // Do not activate module if error 'not allowed' returned when loading module SQL queries (the _load_table run sql with run_sql with the error allowed parameter set to 'default')
		}

So all sql files with -mymodule into its name are executed during first module activation.

Reply:
Hi @eldy, i just did it, it seems to work